contracts/extensions/IColonyExtension.sol

```diff
@@ -19,6 +19,7 @@
 pragma solidity 0.8.27;
 pragma experimental ABIEncoderV2;
 import { IBasicMetaTransaction } from "./../common/IBasicMetaTransaction.sol";
+import { IDSAuth } from "./../common/IDSAuth.sol";
```

Comment:
Is this meant to be used for something?

Reply:
It's being used to make sure that the functions implemented via `DSAuth` are commented - by including this, when the contract is flattened, the interface is included, which is what has the comments.

Reply:
Can you add a comment to that effect? Otherwise it just looks like a redundant import